### PR TITLE
Add JSON visualization to Vulnerability Detection inventory flyout 

### LINKED ISSUE #6589: Add JSON visualization to Vulnerability Detection inventory flyout
We need to add a JSON visualization to the Vulnerability Detection inventory flyout. It should be a tab showing the document inspected as a JSON, similar to the new Discover

image

### PR DESCRIPTION
**Pull request by Tostti**
### Description
This PR adds a tab with the JSON view of a document inside the Vulnerability detection inventory flyout.
 
### Issues Resolved
#6589 

### Evidence
image
image

### Test

- Run the 'scripts/vulnerabilities-events-injector/dataInjectScript.py' to inject vulnerabilities to the environment
- Navigate to Vulnerability Detection section, and Inventory tab
- Click on the Inspect details icon of a row, to open the Flyout
- Verify that the flyout contains a `Table` and a `JSON` sections. Verify that the information shown by both tabs is the same, and corresponds to the selected row.

**If no vulnerabilities are shown it may be because the manager has the cluster disabled. Follow this steps to enable it**
- Attach a console to the manager
- Run the following commands
```
sed -i 's/<node>NODE_IP<\/node>/<node>127.0.0.1<\/node>/' /var/ossec/etc/ossec.conf
sed -i 's/<key><\/key>/<key>c98b62a9b6169ac5f67dae55ae4a9088<\/key>/' /var/ossec/etc/ossec.conf
sed -i 's/<disabled>yes<\/disabled>/<disabled>no<\/disabled>/' /var/ossec/etc/ossec.conf 
/var/ossec/bin/wazuh-control restart
```

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by jbiset — DISMISSED** [review]
## CR: :green_circle: 

## Test UI :green_circle: 

### Evidence

image

image

**Comment by Desvelao** [comment]
# Test :green_circle: 

image
image

File: plugins/main/public/components/overview/vulnerabilities/common/components/document_view_table_and_json.tsx
Comment (Desvelao) [inline plugins/main/public/components/overview/vulnerabilities/common/components/document_view_table_and_json.tsx:1]: **suggestion**: we should name the files using `kebab-case`.

Comment (Tostti) [reply plugins/main/public/components/overview/vulnerabilities/common/components/document_view_table_and_json.tsx:1]: Noted. Resolved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/6590/commits/4148e290850de09b7e32efe0dcd58c6698be454a

**Review by Desvelao — APPROVED** [review]
LGTM!

**Review by jbiset — APPROVED** [review]

**Pull request merged**